wsmaster/che-core-api-workspace-shared/src/main/java/org/eclipse/che/api/workspace/shared/Constants.java

```diff
@@ -39,6 +39,17 @@
 
   public static final String CHE_WORKSPACE_AUTO_START = "che.workspace.auto_start";
 
+  /** This attribute describes time when workspace was created. */
```

Comment:
It is not clear these are workspace attributes. Can you add `{@link Workspace#getAttributes}` link to the javadocs?